charts/datadog/README.md

```diff
@@ -693,9 +693,7 @@ helm install <RELEASE_NAME> \
 | datadog.systemProbe.enableConntrack | bool | `true` | Enable the system-probe agent to connect to the netlink/conntrack subsystem to add NAT information to connection data |
 | datadog.systemProbe.enableDefaultKernelHeadersPaths | bool | `true` | Enable mount of default paths where kernel headers are stored |
 | datadog.systemProbe.enableDefaultOsReleasePaths | bool | `true` | enable default os-release files mount |
-| datadog.systemProbe.enableKernelHeaderDownload | bool | `true` | Enable the downloading of kernel headers for runtime compilation of eBPF probes |
 | datadog.systemProbe.enableOOMKill | bool | `false` | Enable the OOM kill eBPF-based check |
-| datadog.systemProbe.enableRuntimeCompiler | bool | `false` | Enable the runtime compiler for eBPF probes |
```

Comment:
Ideally, I would like to find a way to leave these config options in the README in order to provide visibility that they exist, but I'm not sure how to do that when they don't exist by default.

Note that having no value set (i.e. having `enableRuntimeCompiler: <empty>`) in `values.yaml` doesn't work, as there is no way AFAIK for the configmap to differentiate between a nil value and a manually set value of `false`. We need to be able to tell the difference, because if it's nil we don't want to write anything to the config map, but if it's `false` we do.

Reply:
@brycekahle I found  a helm template function which allows us to check directly whether or not the value has been set, so we can leave these options in our `values` file and still know whether or not to write them to the configmap, which solves this problem